### DISCUSSION TITLE
add C library to VS code

### DISCUSSION THREAD

**Discussion by GastonMelo** [microsoft/vscode-discussions#576] [Extension Development QnA]
Hi to all, Im using the ftd2xx library in  VS code. I already downloaded the library from ftdi web site and follow the instruccion on using in linux (Im using Linux Tumbleweed 64 OS).
I copy the header and libraries to /usr/local/inlcude and /usr/local/lib, i add this paths in the properties.json

«"includePath": [
                "${workspaceFolder}/**",
                "/usr/local/include/**",
                "/usr/local/lib/**",
                "/home/gaston/CProjects/lib/**"
            ],»

i doesn't work. I create a task.json i put this on it:

«
"args": [
                "-std=c99",
                "gcc",
                "${fileDirname}/*.h",
                "${fileDirname}/*.c",
                "-o",
                // Ask msbuild to generate full paths for file names.
                "/property:GenerateFullPaths=true",
                "/t:build",
                // Do not generate summary otherwise it leads to duplicate errors in Problems panel
                "/consoleloggerparameters:NoSummary",
                "-L/usr/local/lib/**",
                "-llibftd2xx.so.1.4.27"
            ],
»
doesn't work either.
I created a folder inside the project and put the lib in there and doesn't work either, only If I compile the project with : 
« gcc ftdi.c lib/libftd2xx.so.1.4.27 -o ftdi»
the program is compiled and work.
Sorry to ask in here but any help will be appreciated
Gaston

**Comment by digitarald** [comment]
Sorry, but these discussions are meant for extension development questions. I recommend to bring this to StackOverflow.
